src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs

```diff
@@ -1172,7 +1172,7 @@ XamarinAndroidApplicationProject CreateMultiDexRequiredApplication (string debug
 		}
 
 		[Test]
-		[Category ("Minor")]
+		[Category ("DotNetIgnore")]
```

Comment:
Given that "main" is now *only* .NET, should we consider just *removing* this test?  I'm guessing this is largely "moot" because multidex is supported in API-21+…

@dellis1972?

Reply:
I've removed this test.